config/rbac/volcano_rbac.yaml

```diff
@@ -0,0 +1,30 @@
+# This file provides RBAC permissions for volcano gang scheduling.
+# To enable volcano gang scheduling, uncomment this file in kustomization.yaml
+# and ensure gangSchedulingManagement.schedulerProvider is set to "volcano" in controller configuration.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: lws-volcano-role
```

Comment:
HI @JesseStutler

If we can avoid creating a new ClusterRole lws-volcano-role here and instead reuse an existing ClusterRole, it would be more consistent with Helm.
Then we can use the patches mechanism in config/default/kustomization.yaml.

Would this be better?

Reply:
If we update role.yaml, in any case this role will be added. But this functionality is optional. So that in my opinion it is better to have it separately.